lib/Controller/RoomController.php

```diff
@@ -834,14 +965,26 @@ protected function formatParticipantList(array $participants, bool $includeStatu
 		return new DataResponse(array_values($results), Http::STATUS_OK, $headers);
 	}
 
+	/**
+	 * Add a participant to a room
+	 *
+	 * @param string $newParticipant New participant
+	 * @param string $source Source of the participant
+	 * @return DataResponse<Http::STATUS_OK, array{type: ?int}, array{}>|DataResponse<Http::STATUS_NOT_FOUND|Http::STATUS_NOT_IMPLEMENTED, array<empty>, array{}>|DataResponse<Http::STATUS_BAD_REQUEST, array{error: ?string}, array{}>
+	 * return DataResponse<Http::STATUS_OK|Http::STATUS_BAD_REQUEST|Http::STATUS_NOT_FOUND|Http::STATUS_NOT_IMPLEMENTED, ?array{type?: int, error?: string}, array{}>
```

Comment:
is this an earlier attempt from the previous line?

Reply:
Oh, probably yes

Reply:
Deduplicated in https://github.com/nextcloud/spreed/pull/10186/commits/59d25a7abcb951db211b6d70369904403a2d18a8